### ISSUE TITLE
ui5-icon does not have aria-label on the internal svg tag

### ISSUE THREAD

**Issue by gurkirpalgill** [SAP/ui5-webcomponents#1681]
**Describe the bug**
ui5-icon does not have aria-label on the internal svg tag

**To reproduce**
See attached screenshot.

![Screen Shot 2020-05-22 at 6 07 04 PM](https://user-images.githubusercontent.com/65705637/82718445-62afd600-9c57-11ea-9f34-2fb89100aa8f.png)

**Comment by ilhan007** [comment]
Hello @gurkirpalgill 

you are looking at the stable version 1.0.0-rc.7, I suggest testing on the following url, which shows the latest development version 0.0.0-d9933bd82 (note the "master" part of the URL):

https://sap.github.io/ui5-webcomponents/master/playground/components/Icon/

When you open the same page, you will notice that the icon does not have an aria-label attribute, but it has now role="presentation" (previously role="img"). 

I tested with Axe and no issue have been found:


<img width="1263" alt="Screenshot 2020-05-23 at 8 46 45 AM" src="https://user-images.githubusercontent.com/15702139/82722742-04740900-9cd2-11ea-98df-19029d85e8e3.png">
